### DISCUSSION TITLE
How to use a msi property in a Feature title and description. The value of the property is being taken from MSI property

### DISCUSSION THREAD

**Discussion by AnjaliMamidi** [wixtoolset/issues#7285] [Questions]
I want to get the MSI property value in the Feature title. But I'm unable to achieve it as I'm seeing the property name within the braces[] in the setup.

I've tried something like this

<Feature Id="Complete" Title="[TEST]" Description="Installation" …> but I just get a feature title [TEST], not the value which I set via MSI property.

But the expected behavior is to get feature title as "TestString" which I'm not observing.

Any support provided would be appreciated!!!

**Comment by BMurri** [comment]
The short answer is that you can't, because the Title field is what MSI calls of type `Text`, which does not process the content to discover embedded property references. It would need to be of type `Formatted` in order to do what you wish.

https://learn.microsoft.com/windows/win32/msi/feature-table
